### PR TITLE
Add more wine types to numerous recipes

### LINKED ISSUE #27565: Different wines not interchangeable in recipes where you would expect them to be
Just noticed that with the exception of making brandy named wines (Riesling, Chardonnay, Cabernet Sauvignon, Pinot Noir) and improvised wines (dandelion wine, pine wine) are not usable in recipes that use wine like sugar, denatured alcohol, unfermented vinegar and ethanol. 

It feels all wines should be interchangeable for at least the etanol and denatured alcohol recipes and the named wines and "cheap wine" should be usable in vinegar, regarding the sugar recipe i'm not sure you could extract sugar from wine in the first place?

Another alternative would of course be to simply remove all these different types of wine and just have a generic wine and add that to the relevant recipes.

### PR DESCRIPTION
**Pull request by Pupsi-Mupsi**
#### Summary

`SUMMARY: Bugfixes "Add more wine types to numerous recipes"`

#### Purpose of change

- Wine should now be more interchangeable - considering its use in recipes like chem_ethanol and denat_alcohol
-> worthy_wine proportion is the same as in the preexisting brandy recipe
- Added missing items from alcohol.json to NC_BARTENDER.json
- Added two new requirements (cheap_wine and worthy_wine)
- Linted recipe_food.json, NC_BARTENDER.json, fuel.json, brewing.json and distill.json.

This closes #27565

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
I think it would be better to add a new requirement like `wines`, add all of the wines there, and replace big lists of wines with this requirement.

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -4114,7 +4114,8 @@
     "components": [
       [
         [ "sweet_fruit_like", 2, "LIST" ],
-        [ "bum_wine", 14 ],
+        [ "cheap_wine", 14, "LIST" ],
+        [ "worthy_wine", 10, "LIST" ],
```
Comment (Reclusive-reptile) [inline data/json/recipes/recipe_food.json]: There are 5g of total carbohydrates in 150ml of wine. Attempting to refine wine into sugar would be extremely inefficient.

**Comment by jbytheway** [comment]
You're changing the whole content of `data/json/recipes/food/brewing.json` and `data/json/recipes/food/distill.json` so they should be removed from the linting blacklist (`json_blacklist`).

**Comment by Pupsi-Mupsi** [comment]
I did not lint just for linting. Here are the changes that really mattered: 
grafik
grafik
grafik
grafik
grafik
grafik
grafik

**Pull request merged**